chainweb.cabal

```diff
@@ -110,7 +110,7 @@ common warning-flags
 
```

Comment:
You need to update the freeze file now.

```
rm cabal.project.freeze
cabal update
cabal freeze
```

Reply:
Just noticed. I am also wondering why there are not non-freeze file builds. It seems that there is much less ci coverage for cabal CI builds than there used to be -- in terms of dependencies, ghc versions, and platforms? Is there any reason for that?

Reply:
I think, that's the reason why the breaking changes in crypton-connection went unnoticed for a while -- preventing testing of latest versions of tis and http related packages.

Reply:
> I think, that's the reason why the breaking changes in crypton-connection went unnoticed for a while -- preventing testing of latest versions of tis and http related packages.

I did notice, and I wrote a PR that already fixed this - I think it got accidentally "reverted" in a bad merge.

There were non-freeze builds until recently, which is when I noticed. Non-freeze builds were causing a lot of churn for working on CI, which is why we disabled them

Reply:
I am not sure if those "cause" churn. I think, they rather "remind" of chores that need to be done anyways. 

Usually, those are simple tasks which can be done quickly, but can become nasty when they start to pile up.

In case those tasks are not simple, we used to add upper bounds in `cabal.project` and considered them as open issues.

Reply:
But, yeah, it's a matter of process. 

I think, there should be some process in place, to catch and track dependency issues early on. Otherwise, upgrades get pushed back which can cause overhead during release preparation and also reduces the time that new versions of dependency (direct or indirect) get tested.